packages/api/package.json

```diff
@@ -37,15 +37,15 @@
     "test:build": "parcel build --no-autoinstall --no-minify --bundle-node-modules -t browser --out-dir ../dist-worker ../src/worker.js",
     "coverage": "yarn run test --coverage",
     "pkg": "pkg --out-path=bin --no-bytecode --public-packages \"*\" --public . --compress GZip ",
-    "pkg:local": "yarn run prepare && yarn run pkg --targets=$(node -p '`node18-${os.platform()}-${os.arch()}`')"
+    "pkg:local": "yarn run prepare && yarn run pkg --targets=$(node -p '`node18-${os.platform()}-${os.arch()}`')",
```

Comment:
Do we still need to keep these?

Reply:
Yeah, we're still planning to use the pkg version as the primary thing in the box (if only because I can't get the embedded frontend working well with esbuild) so it's still useful to be able to build that when necessary. But it takes like ~3min instead of ~3sec for the esbuild, so we'll use esbuild in the box dev environment.